classes/Kohana/Arr.php

```diff
@@ -8,7 +16,7 @@
  * @copyright  (c) 2007-2014 Kohana Team
  * @license    http://kohanaframework.org/license
  */
-abstract class Kohana_Arr {
+class Arr {
```

Comment:
There are no instance methods in `\Kohana\Core\Arr` though, so it doesn't make any difference?